src/main/java/com/github/krzysiekjodlowski/ox/ui/ConsoleUI.java

```diff
@@ -0,0 +1,59 @@
+package com.github.krzysiekjodlowski.ox.ui;
+
+import com.github.krzysiekjodlowski.ox.NumberRange;
+
+import java.io.PrintStream;
+import java.util.Scanner;
+
+/**
+ * {@inheritDoc} The console version.
+ *
+ * @author Krzysztof Jodlowski
+ */
+public class ConsoleUI implements UI<String, Integer> {
+  private final Scanner scanner;
+  private final PrintStream printStream;
```

Comment:
Error handling needs some love. Please add PrintStream for error showing (you can pass System.err from main) 😺.

Reply:
That's done! Basically what I've done is a little walkaround by using the same stream and only changing the color of the String to red.